haskell-src/Concordium/Types/Updates.hs

```diff
@@ -978,6 +988,7 @@ extractKeysIndices p =
         BlockEnergyLimitUpdatePayload{} -> getLevel2KeysAndThreshold asParamConsensusParameters
         FinalizationCommitteeParametersUpdatePayload{} -> getLevel2KeysAndThreshold asPoolParameters
         ValidatorScoreParametersUpdatePayload{} -> getLevel2KeysAndThreshold asPoolParameters
+        CreatePLTUpdatePayload{} -> undefined
```

Comment:
Can you replace this with an error and a message referencing the issue for the implementation?

Reply:
I'll add a message, but I was not able to find an/the issue tracking this.